java-client/src/main/java/org/opensearch/client/opensearch/cat/nodes/NodesRecord.java

```diff
@@ -140,9 +140,13 @@ public class NodesRecord implements JsonpSerializable {
 	@Nullable
 	private final String nodeRole;
 
+	@Deprecated
```

Comment:
nit: I think technically `private` variable or method can be renamed freely without deprecating. 😁

Reply:
I think that is fine if not changed. 😄